### ISSUE TITLE
Pagination for view users page

### ISSUE THREAD

**Issue by BrainMaestro** [SkaeX/FCE_Re#1]
It just lists one long page of results.

**Comment by SkaeX** [comment]
True. I was thinking of using datatables to solve that. Check it out on https://www.datatables.net/

**Comment by BrainMaestro** [comment]
Yeah, I remember you mentioning it before. Looks good
